test/github/models/test_pull_request.py

```diff
@@ -14,6 +14,17 @@ def test_requested_reviewers_github_users_or_teams(self):
         )
         self.assertEqual(["user1", "user2"], pull_request.requested_reviewers())
 
+    def test_requested_reviewers_github_team_subscription_disabled(self):
+        user = build(builder.user().login("user1"))
+        pull_request = build(
+            builder.pull_request()
+            .requested_reviewers([user])
+            .requested_reviewer_team("some_team", ["user1", "user2"])
+        )
+        self.assertEqual(
+            ["user1"], pull_request.requested_reviewers(include_team_members=False)
```

Comment:
I see you're bypassing the feature flag check by testing one level deeper. I think that's fine - but were you hoping to test this another way and ran into issues with the feature flag? (this is related to your "cargo cult" comment in #144

Reply:
My comment on #144 was mostly about how to test/deploy the the flag from a terraform PoV, my manual testing covers the flag worked as expected from the envvar perspective (but I can add a concrete test for that too if you think it's valuable :) )

Reply:
Gotcha. Take a look at https://app.asana.com/0/1149418478823393/815237203241118/f (which you'll also want to update once this is merged, if you want to have Asana's SGTM enable this flag).

Let me know if you'd like further help testing this. I think you mainly want to `export SGTM_FEATURE__DISABLE_GITHUB_TEAM_SUBSCRIPTION=true` before opening up your `ipython` shell